packages/exec/__tests__/exec.test.ts

```diff
@@ -286,6 +286,30 @@ describe('@actions/exec', () => {
     expect(stderrCalled).toBeTruthy()
   })
 
+  it('Handles stdin', async () => {
+    let command: string
+    if (IS_WINDOWS) {
+      command = 'wait-for-input.cmd'
+    } else {
+      command = 'wait-for-input.sh'
```

Comment:
I would like to see a test that shows what an action author would do.  e.g. all js instead of the sh and cmd scripts.

Reply:
I added a js-specific test, although I admit I'm not sure exactly what you're asking. All of these tests spawn a childprocess using exec - there's no "all js". The test spawns a node subprocess with a node file that prints what it gets on stdin.